config/config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  PORT: 3000,
+  DEV_SECRET: 'dev-secret',
+  DATABASE_URL: 'mongodb://localhost:27017/mestodb',
+};
```

Comment:
Можно лучше: использовать `process.env` в конфигурационном файле.
Конфиг у нас представляет обычный JS файл, поэтому здесь можно настраивать значения в зависимости от условий. Например, считать `.env` файл и задать значения для продакшена:
```suggestion
require('dotenv').config();

const isProduction = process.env.NODE_ENV === 'production';

module.exports = {
  PORT: 3000,
  JWT_SECRET: isProduction ? process.env.JWT_SECRET: 'dev-secret',
  DATABASE_URL: 'mongodb://localhost:27017/mestodb',
};
```